### ISSUE TITLE
No longer scrobbling

### ISSUE THREAD

**Issue by owenrenn** [newgiin/cloudplayer-scrobbler#31]
It looks like something has changed on Google Play; noticing on this and a different plugin when looking at debugging that it's pulling the artist info fine but track names aren't being grabbed anymore and therefore we're seeing an error on trying to submit (and consequently no scrobbles)

**Comment by ekovi** [comment]
Hi

I have the same...
@newgiin, will you be able to action this?

Below is the current html source of the 'player' div, shows where's the song title and album stored now.

song title:

``` html
<div id="currently-playing-title" class="tooltip">November Rain</div>
```

album info:

``` html
<div class="player-album" data-type="album" data-id="Bhf23rqxwpelunmn744tltqfygy/Guns+N'+Roses/Use+Your+Illusion+I+(Explicit+Version)" data-navigate="">Use Your Illusion I (Explicit Version)</div>
```

Player div content:

``` html
<div id="player" class="g-section material active" role="navigation" main=""><div id="player_api"></div><div class="player-progress-wrapper"><paper-slider id="material-player-progress" role="slider" tabindex="0" aria-disabled="false" aria-valuemin="0" aria-valuemax="537000" aria-valuenow="98123" class="x-scope paper-slider-0" title="Track progress" value="98123">

    <div id="sliderContainer" class="style-scope paper-slider">

      <div class="bar-container style-scope paper-slider">
        <paper-progress id="sliderBar" aria-hidden="true" class="style-scope paper-slider x-scope paper-progress-0" role="progressbar" value="98123" aria-valuenow="98123" aria-valuemin="0" aria-valuemax="537000" aria-disabled="false" style="touch-action: none;">
    <div id="progressContainer" class="style-scope paper-progress">
      <div id="secondaryProgress" class="style-scope paper-progress" style="transform: scaleX(0);" hidden=""></div>
      <div id="primaryProgress" class="style-scope paper-progress" style="transform: scaleX(0.182724);"></div>
    </div>
  </paper-progress>
      </div>

      <template is="dom-if" class="style-scope paper-slider"></template>

      <div id="sliderKnob" class="center-justified center horizontal layout style-scope paper-slider" style="touch-action: none; left: 18.2724%;">
          <div id="sliderKnobInner" class="style-scope paper-slider" value="98123"></div>
      </div>
    </div>
    <template is="dom-if" class="style-scope paper-slider"></template>
  </paper-slider></div><div id="material-player-left-wrapper"><div class="player-left"><div id="playerSongInfo"><div class="image-wrapper"><img id="playerBarArt" src="https://lh3.googleusercontent.com/GvATmB1Y3XK9qXgb836MhoT_cosHUTWL4lK8PF8IpwZTNUSZe4mqqFaZfAuIdkzVMPbs2m8z=s90-c-e100" alt=""><div class="hover-overlay"><div id="hover-icon" class="hover-icon"></div></div></div><div class="now-playing-info-wrapper"><div class="now-playing-info-content"><div id="currently-playing-title" class="tooltip">November Rain</div><div class="currently-playing-details"><div id="player-artist" class="player-artist" data-type="artist" data-id="A3h4chznmsiouuyxfwme5kyktzy/Guns+N'+Roses" data-navigate="">Guns N' Roses</div><div class="player-dash">&nbsp;-&nbsp;</div><div class="player-album" data-type="album" data-id="Bhf23rqxwpelunmn744tltqfygy/Guns+N'+Roses/Use+Your+Illusion+I+(Explicit+Version)" data-navigate="">Use Your Illusion I (Explicit Version)</div></div></div><div class="now-playing-actions "><paper-icon-button data-id="now-playing-menu" icon="more-vert" title="More options for November Rain" aria-label="More options for November Rain" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><div class="player-rating-container"><div class="rating-container materialThumbs" style="visibility: visible;"><paper-icon-button icon="sj:thumb-up-outline" data-rating="5" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Undo thumb-up" aria-label="Undo thumb-up">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M1 21h4V9H1v12zm22-11c0-1.1-.9-2-2-2h-6.31l.95-4.57.03-.32c0-.41-.17-.79-.44-1.06L14.17 1 7.59 7.59C7.22 7.95 7 8.45 7 9v10c0 1.1.9 2 2 2h9c.83 0 1.54-.5 1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-1.91l-.01-.01L23 10z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button icon="sj:thumb-down-outline" data-rating="1" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Thumb-down" aria-label="Thumb-down">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g transform="translate(2.000000, 3.000000)" class="style-scope iron-icon">
  <path d="M12.9,0H4C3.2,0,2.5,0.5,2.2,1.2l-3,7.3C-0.9,8.7-1,8.9-1,9.2v2 c0,1.1,0.9,2,2,2h6.3l-1,4.7v0.3c0,0.4,0.2,0.8,0.4,1.1l0,0C7.3,20,8.2,20,8.8,19.4l5.5-5.7c0.4-0.4,0.6-0.9,0.6-1.4V2 C14.9,0.9,14,0,12.9,0L12.9,0z M12.7,12.6l-3.5,3.6c-0.2,0.2-0.5,0-0.4-0.2l1-4.6H2c-0.6,0-1-0.5-1-1V9.4l0-0.1l2.7-6.6 C3.9,2.2,4.3,2,4.7,2L12,2c0.5,0,1,0.5,1,1v8.8C12.9,12.1,12.8,12.4,12.7,12.6L12.7,12.6z" class="style-scope iron-icon"></path>
  <path d="M17,0h4v12h-4V0z" class="style-scope iron-icon"></path>
</g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button icon="remove-circle-outline" data-rating="0" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M7 11v2h10v-2H7zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button></div></div></div></div></div></div></div><div class="material-player-middle"><paper-icon-button data-id="rewind-30" icon="av:replay-30" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" style="display: none;">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><defs class="style-scope iron-icon"><path id="a" d="M0 0h24v24H0V0z" class="style-scope iron-icon"></path></defs><clipPath id="b" class="style-scope iron-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#a" overflow="visible" class="style-scope iron-icon"></use></clipPath><path d="M12 5V1L7 6l5 5V7c3.3 0 6 2.7 6 6s-2.7 6-6 6-6-2.7-6-6H4c0 4.4 3.6 8 8 8s8-3.6 8-8-3.6-8-8-8zm-2.4 8.5h.4c.2 0 .4-.1.5-.2s.2-.2.2-.4v-.2s-.1-.1-.1-.2-.1-.1-.2-.1h-.5s-.1.1-.2.1-.1.1-.1.2v.2h-1c0-.2 0-.3.1-.5s.2-.3.3-.4.3-.2.4-.2.4-.1.5-.1c.2 0 .4 0 .6.1s.3.1.5.2.2.2.3.4.1.3.1.5v.3s-.1.2-.1.3-.1.2-.2.2-.2.1-.3.2c.2.1.4.2.5.4s.2.4.2.6c0 .2 0 .4-.1.5s-.2.3-.3.4-.3.2-.5.2-.4.1-.6.1c-.2 0-.4 0-.5-.1s-.3-.1-.5-.2-.2-.2-.3-.4-.1-.4-.1-.6h.8v.2s.1.1.1.2.1.1.2.1h.5s.1-.1.2-.1.1-.1.1-.2v-.5s-.1-.1-.1-.2-.1-.1-.2-.1h-.6v-.7zm5.7.7c0 .3 0 .6-.1.8l-.3.6s-.3.3-.5.3-.4.1-.6.1-.4 0-.6-.1-.3-.2-.5-.3-.2-.3-.3-.6-.1-.5-.1-.8v-.7c0-.3 0-.6.1-.8l.3-.6s.3-.3.5-.3.4-.1.6-.1.4 0 .6.1.3.2.5.3.2.3.3.6.1.5.1.8v.7zm-.8-.8v-.5c0-.1-.1-.2-.1-.3s-.1-.1-.2-.2-.2-.1-.3-.1-.2 0-.3.1l-.2.2s-.1.2-.1.3v2s.1.2.1.3.1.1.2.2.2.1.3.1.2 0 .3-.1l.2-.2s.1-.2.1-.3v-1.5z" clip-path="url(#b)" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button data-id="repeat" icon="av:repeat" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Repeat Off. Click to change Repeat mode." aria-label="Repeat Off. Click to change Repeat mode." style="display: none;">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M7 7h10v3l4-4-4-4v3H5v6h2V7zm10 10H7v-3l-4 4 4 4v-3h12v-6h-2v4z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button data-id="rewind" icon="av:skip-previous" title="Previous song" aria-label="Previous song" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M6 6h2v12H6zm3.5 6l8.5 6V6z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button data-id="play-pause" icon="sj:play-circle-fill" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0 playing" title="Pause" aria-label="Pause">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-1 14H9V8h2v8zm4 0h-2V8h2v8z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button data-id="forward" icon="av:skip-next" title="Next song" aria-label="Next song" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M6 18l8.5-6L6 6v12zM16 6v12h2V6h-2z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button data-id="shuffle" icon="av:shuffle" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Shuffle songs" aria-label="Shuffle songs" style="display: none;">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M10.59 9.17L5.41 4 4 5.41l5.17 5.17 1.42-1.41zM14.5 4l2.04 2.04L4 18.59 5.41 20 17.96 7.46 20 9.5V4h-5.5zm.33 9.41l-1.41 1.41 3.13 3.13L14.5 20H20v-5.5l-2.04 2.04-3.13-3.13z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><paper-icon-button data-id="forward-30" icon="av:forward-30" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" style="display: none;">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><defs class="style-scope iron-icon"><path id="a" d="M24 24H0V0h24v24z" class="style-scope iron-icon"></path></defs><clipPath id="b" class="style-scope iron-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#a" overflow="visible" class="style-scope iron-icon"></use></clipPath><path d="M9.6 13.5h.4c.2 0 .4-.1.5-.2s.2-.2.2-.4v-.2s-.1-.1-.1-.2-.1-.1-.2-.1h-.5s-.1.1-.2.1-.1.1-.1.2v.2h-1c0-.2 0-.3.1-.5s.2-.3.3-.4.3-.2.4-.2.4-.1.5-.1c.2 0 .4 0 .6.1s.3.1.5.2.2.2.3.4.1.3.1.5v.3s-.1.2-.1.3-.1.2-.2.2-.2.1-.3.2c.2.1.4.2.5.4s.2.4.2.6c0 .2 0 .4-.1.5s-.2.3-.3.4-.3.2-.5.2-.4.1-.6.1c-.2 0-.4 0-.5-.1s-.3-.1-.5-.2-.2-.2-.3-.4-.1-.4-.1-.6h.8v.2s.1.1.1.2.1.1.2.1h.5s.1-.1.2-.1.1-.1.1-.2v-.5s-.1-.1-.1-.2-.1-.1-.2-.1h-.6v-.7zm5.7.7c0 .3 0 .6-.1.8l-.3.6s-.3.3-.5.3-.4.1-.6.1-.4 0-.6-.1-.3-.2-.5-.3-.2-.3-.3-.6-.1-.5-.1-.8v-.7c0-.3 0-.6.1-.8l.3-.6s.3-.3.5-.3.4-.1.6-.1.4 0 .6.1.3.2.5.3.2.3.3.6.1.5.1.8v.7zm-.9-.8v-.5s-.1-.2-.1-.3-.1-.1-.2-.2-.2-.1-.3-.1-.2 0-.3.1l-.2.2s-.1.2-.1.3v2s.1.2.1.3.1.1.2.2.2.1.3.1.2 0 .3-.1l.2-.2s.1-.2.1-.3v-1.5zM4 13c0 4.4 3.6 8 8 8s8-3.6 8-8h-2c0 3.3-2.7 6-6 6s-6-2.7-6-6 2.7-6 6-6v4l5-5-5-5v4c-4.4 0-8 3.6-8 8z" clip-path="url(#b)" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button></div><div id="material-player-right-wrapper"><div id="volume"><iron-icon id="material-volume-low" icon="av:volume-down" class="x-scope iron-icon-1"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M18.5 12c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM5 9v6h4l5 5V4L9 9H5z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon><paper-slider id="material-vslider" min="0" max="100" step="5" title="Volume" role="slider" tabindex="0" aria-disabled="false" value="40" aria-valuemin="0" aria-valuemax="100" aria-valuenow="40" class="x-scope paper-slider-0">

    <div id="sliderContainer" class="style-scope paper-slider">

      <div class="bar-container style-scope paper-slider">
        <paper-progress id="sliderBar" aria-hidden="true" class="style-scope paper-slider x-scope paper-progress-1" role="progressbar" value="40" aria-valuenow="40" aria-valuemin="0" aria-valuemax="100" aria-disabled="false" style="touch-action: none;">
    <div id="progressContainer" class="style-scope paper-progress">
      <div id="secondaryProgress" class="style-scope paper-progress" hidden="" style="transform: scaleX(0);"></div>
      <div id="primaryProgress" class="style-scope paper-progress" style="transform: scaleX(0.4);"></div>
    </div>
  </paper-progress>
      </div>

      <template is="dom-if" class="style-scope paper-slider"></template>

      <div id="sliderKnob" class="center-justified center horizontal layout style-scope paper-slider" style="touch-action: none; left: 40%;">
          <div id="sliderKnobInner" class="style-scope paper-slider" value="40"></div>
      </div>
    </div>
    <template is="dom-if" class="style-scope paper-slider"></template>
  </paper-slider><iron-icon id="material-volume-high" icon="av:volume-up" class="x-scope iron-icon-1"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M3 9v6h4l5 5V4L7 9H3zm13.5 3c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM14 3.23v2.06c2.89.86 5 3.54 5 6.71s-2.11 5.85-5 6.71v2.06c4.01-.91 7-4.49 7-8.77s-2.99-7.86-7-8.77z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon><iron-icon id="material-volume-indicator" icon="av:volume-down" class="x-scope iron-icon-1"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M18.5 12c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM5 9v6h4l5 5V4L9 9H5z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon></div><paper-icon-button data-id="cast" icon="hardware:cast" title="Cast" aria-label="Cast" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" style="display: none;">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M21 3H3c-1.1 0-2 .9-2 2v3h2V5h18v14h-7v2h7c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM1 18v3h3c0-1.66-1.34-3-3-3zm0-4v2c2.76 0 5 2.24 5 5h2c0-3.87-3.13-7-7-7zm0-4v2c4.97 0 9 4.03 9 9h2c0-6.08-4.93-11-11-11z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><div id="buy_button_player" style="display:none;"><a href="https://play.google.com/store/music/album?id=Bhf23rqxwpelunmn744tltqfygy&amp;authuser=0" target="_blank"><paper-button class="material-primary x-scope paper-button-0" role="button" tabindex="0" animated="" aria-disabled="false">


    Buy
  </paper-button></a></div><paper-icon-button id="queue" data-id="queue" data-type="ap" icon="av:queue-music" aria-haspopup="true" aria-controls="queue-overlay" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0 target" title="Open queue" aria-label="Open queue" aria-expanded="false">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M15 6H3v2h12V6zm0 4H3v2h12v-2zM3 16h8v-2H3v2zM17 6v8.18c-.31-.11-.65-.18-1-.18-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3V8h3V6h-5z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button><div class="player-top-right-items"><div class="time-label"><span id="time_container_current" aria-label="Current track time: 1 minute 39 seconds">1:39</span> / <span id="time_container_duration" aria-label="Track length: 8 minutes 57 seconds">8:57</span></div><paper-icon-button data-id="show-miniplayer" icon="open-in-new" title="Show mini player" aria-label="Show mini player" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0">

    <iron-icon id="icon" class="style-scope paper-icon-button x-scope iron-icon-0"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z" class="style-scope iron-icon"></path></g></svg>
  </iron-icon>
  </paper-icon-button></div></div></div>
```

If you need more info, pls let me know.
Thank you for your work!

**Comment by ekovi** [comment]
Hi @newgiin, I did some debugging of the extension...

In the _**contentscript.js**_ three changes would be required. I've done them locally, made surface tests and looks as if it's working.

a) GoogleMusicParser.prototype._get_song_title function change to fix title issue (and scrobbling)

``` javascript
/**
 * Get current song title
 *
 * @return Song title
 */
GoogleMusicParser.prototype._get_song_title = function() {
    // the text inside the div located inside element with id="playerSongTitle"
    //Originally it was: return $("#player-song-title").text();
    //changed to
    return $("#currently-playing-title").text();
};
```

b) GoogleMusicParser.prototype._get_song_cover function change to get the artwork working

``` javascript
/**
 * Get current song artwork
 *
 * @return Image URL or default artwork
 */
GoogleMusicParser.prototype._get_song_cover = function() {
    //originally was: var albumImg = $("#playingAlbumArt").attr("src");
    //changed to:
    var albumImg = $("#playerBarArt").attr("src");
    if (albumImg)
        return albumImg;
    return null;
};
```

c) GoogleMusicParser.prototype._get_song_album function change to print the album title

``` javascript
/**
 * Get current song album name
 *
 * @return Album name or null
 */
GoogleMusicParser.prototype._get_song_album = function() {
    //originally was: return $("#playerSongInfo .player-artist-album-wrapper .player-album").text();
    //changed to:
    return $(".now-playing-info-wrapper .now-playing-info-content .player-album").text();
};
```

**Comment by ghost** [comment]
It stopped this morning for me too. Thanks @ekovi for your working fix! :-)

**Comment by MathieuLLF** [comment]
I know @ekovi is working on it, but just figured I'd mention it's not working for me either.

Thanks for the effort! :)

**Comment by ekovi** [comment]
@MathieuLLF do not get me wrong. The only I did was to debug the problem, make changes on my local machine and send the possible solution as a comment to the reported issue. I even do not try to make the actual changes in the code on github and wouldn't know how to build or even do not think I have sufficient access to publish it as a new, fixed version of the extension :(

**Comment by MathieuLLF** [comment]
@ekovi oops I misread someone elses post and assumed you were the dev ;) 

Cool on you to have figure it out for your machine though!

**Comment by newgiin** [comment]
Hey all,

Sorry for late response, I'll pull in the fix.

**Comment by newgiin** [comment]
Fixed with 0e000304f07d123fb533db67247e43d561d00950, I pushed version 1.6.8 to the Play Store, takes about an hour to publish. Once it is, it should update automatically, or you can force by going to chrome://extensions, checking "Developer mode" in the upper right, and clicking "Update extensions now".
